### DISCUSSION TITLE
很好的翻译插件，有个小问题想请教一下~

### DISCUSSION THREAD

**Discussion by RorschachY** [chunibyocola/sc-translator-crx#108] [Q&A] [answered]
在翻译文献的时候因为有引文的角标超链接，有时候会一句话的后面就直接翻译，但是这句话并不完整。也就是如果一个引用的角标出现在一句话的中间的时候，这句话可能分成两个部分翻译，这个后面能对文献格式的翻译做一些适配吗？~

**Comment by chunibyocola** [comment, marked as answer]
根据你在下面回答的截图和两个网址，我的发现是：
- 在网址 1 中，截断句子的不是角标，而是不按文章格式的链接`FIG.1`，这个链接使用的是 CSS 中的`inline-block`，而在文章中的链接应该用`inline`包裹。
- 在网址 2 中，截断句子的也同样不是角标，而是它在角标后面使用了一个隐藏的 div 元素，此 div 元素包含了引用的信息，正是此方式导致了截断。

以上两个网址的页面翻译虽然会发生句子的截断，但都与角标无关，也不是网页翻译的逻辑有问题。严格来说是这两个网站都不算是正常的文章（对网页来说），虽然它们的文章在肉眼上看来与其他文章没什么不同，但在代码层面上还是不一样的。

但是我注意到这些文章的角标都会使用相同的元素来包裹，也许我会在未来版本的页面翻译中忽略此元素来让此类文章的可读性更高。

**Reply by RorschachY** [reply]
好的，谢谢~~

**Comment by RorschachY** [comment]
![image](https://user-images.githubusercontent.com/54018729/210033063-92d4d070-6c41-4e41-b206-499fe57966b3.png)
![image](https://user-images.githubusercontent.com/54018729/210033196-7a031cb8-529d-442f-827d-78fd853db36e.png)
就像这种有角标的情况，会把一段话切割成两部分。
网址1：https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8177722/
网站2：https://www.jto.org/article/S1556-0864(18)33501-9/fulltext

**Reply by chunibyocola** [reply]
感谢你对问题的补充，我在上面的回答中补充了说明。
